rustls-cert-gen/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition.workspace = true
 keywords.workspace = true
 
 [dependencies]
-rcgen = { path = "../rcgen", default-features = false, features = ["pem"] }
+rcgen = { path = "../rcgen", default-features = false, features = ["pem", "aws_lc_rs"] }
```

Comment:
I think we should use `ring` as the default for now and consider a project-wide switch at some point in the future.